src/App.tsx

```diff
@@ -21,6 +22,7 @@ function App() {
 						<PublicRoute path="/login" exact component={LoginPage} />
 						<PublicRoute path="/verify" exact component={VerifyEmailPage} />
 						<ProtectedRoute path="/account" exact component={AccountSettingsPage} />
+						<PublicRoute path="/event/:id?" exact component={EventPage}/>
```

Comment:
This should actually be a ProtectedRoute, users aren't allowed to fetch any form of data from the API unless they are authenticated.

Reply:
yes i made it public for now because i couldn't view it before